tests/backup/backup_helper.go

```diff
@@ -1218,85 +1089,104 @@ func DeletePodWithLabelInNamespace(namespace string, label map[string]string) er
 }
 
 // backupSuccessCheck inspects backup task
-func backupSuccessCheck(backupName string, orgID string, retryDuration int, retryInterval int, ctx context.Context) (bool, error) {
-	backupDriver := Inst().Backup
+func backupSuccessCheck(backupName string, orgID string, retryDuration time.Duration, retryInterval time.Duration, ctx context.Context) error {
 	if retryDuration == 0 {
-		retryDuration = maxWaitPeriodForBackupCompletionInMinutes
+		retryDuration = maxWaitPeriodForBackupCompletionInMinutes * time.Minute
 	}
 	if retryInterval == 0 {
-		retryInterval = 30
+		retryInterval = 30 * time.Second
 	}
-	backupSuccessCheck := func() (interface{}, bool, error) {
-		bkpUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
-		if err != nil {
-			return "", false, err
-		}
-		backupInspectRequest := &api.BackupInspectRequest{
-			Name:  backupName,
-			Uid:   bkpUid,
-			OrgId: orgID,
-		}
-		resp, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
+	bkpUid, err := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+	if err != nil {
+		return err
+	}
+	backupInspectRequest := &api.BackupInspectRequest{
+		Name:  backupName,
+		Uid:   bkpUid,
+		OrgId: orgID,
+	}
+	statusesExpected := [...]api.BackupInfo_StatusInfo_Status{
+		api.BackupInfo_StatusInfo_Success,
+	}
+	statusesUnexpected := [...]api.BackupInfo_StatusInfo_Status{
+		api.BackupInfo_StatusInfo_Invalid,
+		api.BackupInfo_StatusInfo_Aborted,
+		api.BackupInfo_StatusInfo_Failed,
+	}
+	_backupSuccessCheck := func() (interface{}, bool, error) {
```

Comment:
golint may complain about having leading underscores. How about naming it `backupSuccessCheckFunc` ?

Reply:
Renamed it to backupSuccessCheckFunc.